Dockerfile

```diff
@@ -0,0 +1,26 @@
+# Generated by https://smithery.ai. See: https://smithery.ai/docs/config#dockerfile
+# syntax=docker/dockerfile:1.4
+
+# Build stage
+FROM node:lts-alpine AS builder
+WORKDIR /app
+
+# Install dependencies
+COPY package.json package-lock.json ./
+RUN npm ci --ignore-scripts
+
+# Copy source and build
+COPY . .
+RUN npm run build
+
+# Production stage
+FROM node:lts-alpine AS runtime
+WORKDIR /app
+
+# Copy built artifacts and production dependencies
+COPY --from=builder /app/dist ./dist
+COPY package.json package-lock.json ./
+RUN npm ci --only=production --ignore-scripts
+
+# Default command to start the MCP server
+ENTRYPOINT ["node", "dist/index.js"]
```

Comment:
How are env vars passed on to this server ?

Reply:
It's passed in smithery.yaml